entrypoint.js

```diff
@@ -1,132 +1,208 @@
 /* eslint-disable camelcase */
 
-const {Toolkit} = require('actions-toolkit');
+const fs = require('fs');
+const path = require('path');
+const core = require('@actions/core');
+const exec = require('@actions/exec');
+const github = require('@actions/github');
+
+const workspace = process.env.GITHUB_WORKSPACE;
+const xoPath = path.join(workspace, 'node_modules', '.bin', 'xo');
+
+const fixXoForChildProcess = async () => {
+  const cliMainPath = path.join(path.dirname(fs.realpathSync(xoPath)), 'cli-main.js')
+  fs.writeFileSync(cliMainPath, fs.readFileSync(cliMainPath).toString().replace('process.exit(report.errorCount === 0 ? 0 : 1);', 'process.exitCode = (report.errorCount === 0 ? 0 : 1);'));
+};
```

Comment:
Not sure what this is used for?
@OmgImAlexis, can you explain, please?

Reply:
It's a patch for this PR.

https://github.com/xojs/xo/pull/405

Reply:
Cool. 
Can you add a comment that it is a workaround for the linked PR and could be removed if it gets merged, please.